src/SAWScript/Proof.hs

```diff
@@ -228,20 +335,203 @@ propToPredicate sc (Prop goal) =
 -- implies the conclusion.
 data ProofState =
   ProofState
-  { psGoals :: [ProofGoal]
-  , psConcl :: ProofGoal
-  , psStats :: SolverStats
-  , psTimeout :: Maybe Integer
+  { _psGoals :: [ProofGoal]
+  , _psConcl :: ProofGoal
+  , _psStats :: SolverStats
```

Comment:
Similar question here as with the `Theorem` type: `SolverStats` seems like a more limited view of `Evidence`; is it redundant?